features/org.hibernate.search.eclipse.feature/feature.properties

```diff
@@ -0,0 +1,27 @@
+###############################################################################
+# Copyright (c) 2010-2012 Red Hat, Inc. and others.
```

Comment:
+1. Also there're some information in several places about the license and vendors. I'm not sure if i filled all of them correctly.